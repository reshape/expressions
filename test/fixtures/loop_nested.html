<p>x</p>
<each loop='item_category, key in items'>
  <p>{{ key }}: {{ JSON.stringify(item_category) }}</p>
  <each loop='item of item_category'>
    <p>{{ item }}</p>
  </each>
</each>
<p>x</p>
