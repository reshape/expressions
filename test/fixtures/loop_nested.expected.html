<p>x</p>

  <p>c1: [1,2,3]</p>
  
    <p>1</p>
  
    <p>2</p>
  
    <p>3</p>
  

  <p>c2: [4,5,6]</p>
  
    <p>4</p>
  
    <p>5</p>
  
    <p>6</p>
  

<p>x</p>